src/js/view/controlbar.js

```diff
@@ -347,14 +350,17 @@ define([
             }
         },
         onElapsed: function(model, val) {
-            var elapsedTime;
+            var elapsedTime,
+                countdownTime;
             var duration = model.get('duration');
             if (model.get('streamType') === 'DVR') {
-                elapsedTime = '-' + utils.timeFormat(-duration);
+                elapsedTime = countdownTime = '-' + utils.timeFormat(-duration);
             } else {
                 elapsedTime = utils.timeFormat(val);
+                countdownTime = ((duration - val > 1) ? '-' : '') + utils.timeFormat(duration - val);
```

Comment:
I wouldn't prefix the countdown with `-`. This may get confusing with how we display time in DVR mode. Users will realize it's a countdown after a couple changes of the remaining time.